### PR TITLE
Create Mindustry Defence category

### PR DESCRIPTION
**Pull request by osp54**
End add new server The Siege, it is not working now, but it will work soon

### CODE REVIEW

File: servers_v6.json [hunk 1]
```diff
@@ -102,5 +102,9 @@
   {
     "name": "TSR", 
     "address": ["fifr2.quackhost.uk:26627"]
+  },
+  {
+    "name": "Mindustry Defence"
```
Comment (Quezler) [inline servers_v6.json:107]:
```suggestion
    "name": "Mindustry Defence",
```

**Comment by Darkness6030** [comment]
Firstly you need run the server :/

**Comment by DeltaNedas** [comment]
1. run the server
2. get a dns bruh

**Comment by osp54** [comment]
> 1. run the server
> 2. get a dns bruh

🤦

**Comment by Darkness6030** [comment]
Server is running ~~to Brazil~~

**Comment by Darkness6030** [comment]
I'd like to call this gamemode "The Siege"

**Comment by osp54** [comment]
@Anuken , is it possible merge is this pull now? New gamemode "The Siege" Already working.

**Review by Darkness6030 — APPROVED** [review]
I approve

**Comment by Anuken** [comment]
image

Why are there two server groups?

**Comment by osp54** [comment]
> image
> 
> Why are there two server groups?

this server was previously in the "MD Community" category , and now, I want to create my own category

**Comment by osp54** [comment]
> image
> 
> Why are there two server groups?

And from the first category, I will remove my server

**Comment by Anuken** [comment]
What is the "MD community" category?

**Comment by osp54** [comment]
> What is the "MD community" category?

This used to mean as MinDurka Defence Community

**Pull request merged**